src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+const Big = require('big.js');
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const nOne = new Big(numberOne);
+  const nTwo = new Big(numberTwo);
+
+  switch (operation) {
+    case '-':
+      return nOne.minus(nTwo);
+    case '+':
+      return nOne.plus(nTwo);
+    case 'x':
+      return nOne.times(nTwo);
+    case '÷':
+      return nOne.div(nTwo);
```

Comment:
dont forget to manage DIVISION BY ZERO on next milestone, else it will return an error